packages/devtools_shared/pubspec.yaml

```diff
@@ -11,4 +11,7 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
+  path: ^1.8.0
```

Comment:
should we be able to remove these deps from the devtools_server pubspec now?

Reply:
I'll verify which deps are actually necessary now.

Reply:
I was able to remove the dep on package:usage.